src/components/Button/Loader/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Dot, LoaderWrapper, ContentWrapper, DotsWrapper } from "./styles";
+import { BUTTON_VARIANTS } from "../../constants";
+
+const Loader = ({ children, loading, variant }) => (
```

Comment:
Should `Loader` be placed under `Button` component? What do you think to have it under independent LoaderDot/ like LoaderRing/ already presented in Aurora?

Reply:
From my point of view the Loader cannot exist by itself and should be used as a part of bigger component. Also, the Button is the only component which requires these dots. Could be discussed further if needed.

Reply:
Discussed.